### DISCUSSION TITLE
Run the script in nodejs but can send multiple message

### DISCUSSION THREAD

**Discussion by khalifahaqi** [rigwild/discord-self-bot-console#58] [Q&A]
just like the title stated. is it possible to send message multiple time. i am thinking of put a new command under the first send message command, and between first and second command i would like it to have 10 seconds delay. but seriously your code is amazing. i am overwhelmed. ty for the code rigwild

and what should i add if i want the message to be sent at random times like. the message will send in exactly 1min but is it possible if i add

min = 1 min
max = 1.5 min
and it will randolmy choose between those time when it will send message

**Comment by DeadLyBro** [comment]
Ref to Rigwild's reply: https://github.com/rigwild/discord-self-bot-console/discussions/48#discussioncomment-4232515

**Reply by khalifahaqi** [reply]
Thank you for the random time message, i never know we could do it like that.BTW I need to make my sentence more clearly. I typed it when I am sleepy LOL. Well, what I mean is I want the program to be able to send multiple message. each message have their own timer. Let's say for example, first message I want to send "Hi" for every 1 minute, and second message "hello" for every 2 minutes. Is it possible? Sorry and thank you for answering question from this newbie

**Reply by DeadLyBro** [reply]
Yeah it's possible check rigwild's reply: https://github.com/rigwild/discord-self-bot-console/discussions/48#discussioncomment-4232515

**Reply by khalifahaqi** [reply]
But, that way the message will be send at random. I tried to put the send message cmd under the index code. It should be something like this:

Index
my cmd
my cmd

But it didn't work 😅. Right now i am using two device to send the 2 message that I want 🤔

**Reply by DeadLyBro** [reply]
try this;
```js
let message = messages[i]
```
so its not random. 

make sure to reset i, if you want to;  
```js
if ( i >= messages.length ) {
    i = 0;
    await delay(10 * 1000 + Math.floor(Math.random() * 5 * 1000)); // its delay you can change.
}
```
